<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BattleFiled</class>
 <widget class="QWidget" name="BattleFiled">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="BattleType1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>610</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BattleType2">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>610</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Duel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Save">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>610</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Save&amp;Exit</string>
   </property>
  </widget>
  <widget class="QListWidget" name="PokeList">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>256</width>
     <height>351</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="Battle1">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>610</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>LevelUp</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Back">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>610</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="Pic">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>130</y>
     <width>241</width>
     <height>231</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>351</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Choose Enemy</string>
   </property>
  </widget>
  <widget class="QLabel" name="Shower">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>400</y>
     <width>291</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
